api.coffee

```diff
@@ -75,6 +75,9 @@ module.exports = (API_TOKEN, HOOK_URL)->
         "@#{data['users.simple'][userId]}"
       .replace /<(\S*)>/g, (match, link)->
         link
+      .replace /&amp;/g, "&"
```

Comment:
Slack's message formatting isn't the same as html encoding. Slack escapes only three characters: &, < and >.
https://api.slack.com/docs/formatting